consumer.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from configman import Namespace, ConfigurationManager
+from kombu import Connection
+
+from signingworker.worker import SigningConsumer
+
+log = logging.getLogger(__name__)
+
+
+def main(config):
+    queue_name = 'queue/{}/{}'.format(config.pulse_user, config.worker_type)
```

Comment:
I'm not really sure what the best practices for queue names are, but you might want to include "pending" in here somewhere, since a single worker type has many different types of events. I'm fine either way, just something to think about.